capstone_project/src/test/java/com/google/sps/SchoolTest.java

```diff
@@ -0,0 +1,66 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+/** Class that tests the basic 'get' methods of School objects. */
+@RunWith(JUnit4.class)
+public final class SchoolTest {
+
+  private final ArrayList<String> uciNews =
+      new ArrayList<String>(
```

Comment:
nit: iirc this line can go on the line above and the lines below should be 1 tab to the left

Reply:
oh, I see that you reformatted for linter. if it auto generated the tabbing, don't worry about this comment + just resolve